test/CMakeLists.txt

```diff
@@ -162,44 +170,50 @@ set_tests_properties(${_stopwatch} PROPERTIES LABELS "nomemcheck")
 #-----------------------------------------------------------------------------#
 # ORANGE TESTS
 #-----------------------------------------------------------------------------#
-add_library(CeleritasOrangeTest
+
+add_library(testcel_orange
   orange/OrangeGeoTestBase.cc
 )
-celeritas_target_link_libraries(CeleritasOrangeTest
-  PRIVATE Celeritas::Test
-  PUBLIC Celeritas::Core
+celeritas_target_link_libraries(testcel_orange
+  PRIVATE Celeritas::testcel_harness Celeritas::orange
 )
 
-celeritas_setup_tests(SERIAL PREFIX orange LINK_LIBRARIES CeleritasOrangeTest)
+celeritas_setup_tests(SERIAL PREFIX orange
+  LINK_LIBRARIES testcel_orange Celeritas::orange
+)
 
-# Base-level
+#-----------------------------------------------------------------------------#
+# Base
 celeritas_add_test(orange/BoundingBox.test.cc)
+celeritas_add_test(orange/Orange.test.cc)
 
+#-------------------------------------#
 # Surfaces
+set(CELERITASTEST_PREFIX orange/surf)
 celeritas_add_test(orange/surf/detail/QuadraticSolver.test.cc)
 celeritas_add_test(orange/surf/CylCentered.test.cc)
 celeritas_add_test(orange/surf/GeneralQuadric.test.cc)
 celeritas_add_test(orange/surf/PlaneAligned.test.cc)
 celeritas_add_test(orange/surf/Sphere.test.cc)
 celeritas_add_test(orange/surf/SphereCentered.test.cc)
-celeritas_device_test(orange/surf/SurfaceAction)
+celeritas_add_device_test(orange/surf/SurfaceAction)
+
 
+#-------------------------------------#
 # Universe details
+set(CELERITASTEST_PREFIX orange/univ)
 celeritas_add_test(orange/univ/detail/LogicEvaluator.test.cc)
 celeritas_add_test(orange/univ/detail/LogicStack.test.cc)
 celeritas_add_test(orange/univ/detail/SurfaceFunctors.test.cc)
-
-### Tests that require a geometry input ###
-
 celeritas_add_test(orange/univ/detail/SenseCalculator.test.cc)
 celeritas_add_test(orange/univ/VolumeView.test.cc)
-celeritas_device_test(orange/univ/SimpleUnitTracker)
-celeritas_add_test(orange/Orange.test.cc)
+celeritas_add_device_test(orange/univ/SimpleUnitTracker)
 
 #-----------------------------------------------------------------------------#
 # CELERITAS TESTS
 #-----------------------------------------------------------------------------#
-add_library(CeleritasTest
+
+add_library(testcel_celeritas
```

Comment:
Why not
```suggestion
add_library(celeritas_testbase
```
?

Reply:
(I didn't see your follow-on https://github.com/celeritas-project/celeritas/pull/526#discussion_r1004843186 ... maybe best to try to keep the discussion in one place)

Reply:
I don't think it is worth much more effort but given its name(s) (original, new or even mine), I am still unsure what the purpose of the library is (but that is probably not that important)